Code/GraphMol/FileParsers/test1.cpp

```diff
@@ -353,6 +353,7 @@ void test4() {
   smi = MolToSmiles(*m);
   CHECK_INVARIANT(smi == "c1cc[cH-]c1", smi);
   TEST_ASSERT(m->getConformer().is3D() == false);
+  delete m;
```

Comment:
We should probably make a unique_ptr<ROMol> typedef in ROMol.h and start using that liberally.